warehouse/templates/manage/account.html

```diff
@@ -300,6 +300,21 @@ <h2 class="sub-title">{% trans %}Account details{% endtrans %}</h2>
             Displayed on your <a href="{{ href }}">public profile</a>
           {% endtrans %}
         </p>
+        <label class="form-group__label" for="is_email_private">
+          {% trans %}️Keep my email address private{% endtrans %}
+          {% if save_account_form.name.flags.required %}
+          <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+          {% endif %}
+          {{ save_account_form.is_email_private(class_="form-group__input", **{"aria-describedby":"is-email-private-errors"}) }}
```

Comment:
Let's move this out of the label and into a `<p class="form-group__text">` with some text before it like "Keep my email address private".

Reply:
Done.